tests/plugins/tabletools/cellproperties.js

```diff
@@ -139,6 +140,48 @@
 			} );
 		},
 
+		// (#2084)
+		'test load and update field values (#11)': function() {
+			this.doTest( 'table-12', function( dialog ) {
```

Comment:
Maybe `doTest` function could be reworked to return a function, so we could have few lines less.

It would result in the following:
```
'test load and update field values (#11)': doTest( 'table-12, function( dialog ) {
```